src/lib/parser.js

```diff
@@ -116,6 +116,115 @@ function getFilterForColumnsWithResults(rows, settings, resultsMap) {
 }
 
 /**
+ * Returns the array of indexes of columns that look like keeping player names
+ * by searching for certain header names.
+ *
+ * @param {Array.<Element|HTMLElement>} rows
+ * @param {HighlighterSettings} settings
+ * @returns {Array.<number>}
+ */
+function getIndexesOfColumnsWithNamesByHeaderNames(rows, settings){
+    const regexps = nameHeadersToRegExp(settings.nameColumnHeaders);
+
+    let indexes = [];
```

Comment:
Next 7 lines look like could e replaced with _reduce_ - very similar to lines 74-80